modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/src/main/resources/META-INF/resources/render/form_handlers/metal_js.jspf

```diff
@@ -92,13 +97,35 @@
 	}
 
 	function updateProductFields(productData) {
-		document.querySelector('[data-text-cp-instance-sku]').innerHTML =
-			Liferay.Util.escape(productData.sku) || '';
+		var sku = '';
+		var manufacturerPartNumber = '';
+		var gtin = '';
+		var availability = '';
+		var availabilityEstimate = '';
+		var stockQuantity = '';
+		var subscriptionInfo = '';
+		var deliverySubscriptionInfo = '';
+
+		if (productData) {
+			sku = productData.sku;
+			manufacturerPartNumber = productData.manufacturerPartNumber;
+			gtin = productData.gtn;
```

Comment:
`productData.sku`, `productData.manufacturerPartNumber` and `productData.gtn` can be `undefined` (even if `productData` may not, as an empty object). Ugly as it might look, we'd still need to defend this via, e.g.
```
sku = productData.sku || ''
```
(and so on, like the other keys below) on pain of having an undefined variable re-assignment and then a `"undefined"` as the output of the `Liferay.Util.escape(sku)`.